### PR TITLE
Update prerequisites.md

### PR DESCRIPTION
**Pull request by vaylen**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer/plan-and-deploy/prerequisites.md [hunk 1]
```diff
@@ -74,6 +74,10 @@ You can use any member of the Active Directory domain to prepare Active Director
 1. The computer requires the following software:
 
    a. [.NET Framework 4.8](https://download.visualstudio.microsoft.com/download/pr/014120d7-d689-4305-befd-3cb711108212/0fd66638cde16859462a6243a4629a50/ndp48-x86-x64-allos-enu.exe)
+   
+      > [!NOTE]
+      >
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/plan-and-deploy/prerequisites.md:79]:
```suggestion
      > When installing on Windows Server Core, you must use the "/q" key to install this package. Optionally, you can use "/log [PATH]" for logging.
```

File: Exchange/ExchangeServer/plan-and-deploy/prerequisites.md [hunk 2]
```diff
@@ -74,6 +74,10 @@ You can use any member of the Active Directory domain to prepare Active Director
...
+      > - When installing on Windows Server Core, you must use key "/q" for install this package. Optionaly you can use "/log [PATH]" for logging.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/plan-and-deploy/prerequisites.md:80]:
```suggestion
```

File: Exchange/ExchangeServer/plan-and-deploy/prerequisites.md (hunk 1 above)
Comment (JohanFreelancer9) [reply Exchange/ExchangeServer/plan-and-deploy/prerequisites.md:79]: Mainly suggestions, please confirm technical accuracy.

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Exchange/ExchangeServer/plan-and-deploy/prerequisites.md (hunk 1 above)
Comment (vaylen) [reply Exchange/ExchangeServer/plan-and-deploy/prerequisites.md:79]: Yes, it's right

**Pull request merged**